Create ui-c8y-1019-6-5-allowing-more-links-on-login-page-and-show-it-on-all-public-facing-pages-allows-to-add-more-than-one-link-on-the-login-page-those-links-are-shown-all-public-facing-pages-not-only-on-the-login-page.md (#1740)
SoftwareAG/c8y-docs: content/change-logs/ui-c8y-1019-6-5-allowing-more-links-on-login-page-and-show-it-on-all-public-facing-pages-allows-to-add-more-than-one-link-on-the-login-page-those-links-are-shown-all-public-facing-pages-not-only-on-the-login-page.md

---
date: ""

> title: Allowing more links on the login page and showing it on all public-facing pages

Reviewer: ```suggestion
title: Enabling multiple links on the Login page which are shown on all public-facing pages

```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-53597
version: 1019.6.5
---

> This update enables adding multiple links to the login page by a custom application configuration. These links will be displayed on all public-facing pages of the site, not only on the login page. This provides a consistent experience for users accessing public pages, with easy access to extra helpful links from any page. You can for example add there a privacy notice or a terms and conditions link.

Reviewer: ```suggestion
This update enables adding multiple links to the Login page by a custom application configuration. These links will be displayed on all public-facing pages of the site, not only on the Login page. This provides a consistent experience for users accessing public pages, with easy access to extra helpful links from any page. You can add a privacy notice, for example, or a terms and conditions link.

```

---
date: ""

> title: Allowing more links on login page and show it on all public facing pages

Reviewer: Please use sentence capitalization: Additional links on Login page visible across public pages
Also, I´m not sure if this is what the change is about. Please check.

>  - allows to add more than one link on the login page 

Reviewer: Is this part of the title? Should be removed

 - those links are shown all public facing pages, not only on the login page
product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-53597

> version: 1019.6.5

Reviewer: @janhommes Is that correct? Must be "Login page" (2x)
